runtime/doc/ft_rust.txt

```diff
@@ -276,7 +276,7 @@ g:rust_keep_autopairs_default ~
 
 	Don't override auto-pairs default for the Rust filetype. The default
 	is 0.
- 
+
```

Comment:
I think this blank line should be removed.

Reply:
There are some irregularities regarding separators:

```
<empty line>
<empty line>
<separator>
```

vs.
```
<empty line>
<separator>
```

I will adjust everything to the latter.

At the same time, two blank lines should be kept between each setting for better readability.

Reply:
I'm specifically talking about this one. Please don't change other places.